src/frontend/src/env/tokens.env.ts

```diff
@@ -61,7 +61,7 @@ export const ICP_SYMBOL = 'ICP';
 
 export const ICP_TOKEN_ID = Symbol(ICP_SYMBOL);
 
-export const ICP_TOKEN: Required<IcToken> = {
+export const ICP_TOKEN: Required<Omit<IcToken, keyof TokenAppearance>> & TokenAppearance = {
```

Comment:
cf `Required<Omit<IcToken, keyof TokenAppearance>>` === nice 👍

You need `& TokenAppearance` here?

Reply:
not really, but I would like to have the same "format" in all of the changes i did, so that it is easier to catch them all, once i do the refactoring

Reply:
- PR #1985